HIP/hip-1056.md

```diff
@@ -848,10 +857,33 @@ message CreateAccountOutput {
 
 ```protobuf
 message EthereumOutput {
+   // Preview block streams used these fields for sidecars and verbose
+   // contract function results
+   reserved 2,3,4;
+
     /**
-     * Result details for an EVM transaction execution
+     * If the associated Ethereum transaction required hydrating its
+     * call data from a file, the Keccak256 hash of the transaction.
+     * <p>
+     * If the Ethereum transaction was inline, this field SHALL NOT be set.
```

Comment:
Q: In the case of hydration the File has the transaction which can be used to calculate the hash right?
If so do we need to state the hash?
As part of the execution of the transaction a CN would have to read the File Contents anyway so there's no extra work to obtain the transaction details.

I guess the challenge is for a block stream reader that does not maintain state. 
Is that what this is trying to optimize for?